docker/walrus-antithesis/build-test-config-image/pipeline.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
I think it's helpful to merge this script, so that in the future, we can directly use it to run a local cluster. Can we move it to walrus/scripts dir, since this is not needed for antithesis test?

Reply:
Currently this script is only used to handle the local antithesis-like build/run pipeline. I could move it elsewhere, but I'm not yet convinced that it would improve the legibility/usability of it significantly, while at least having it next to the docker-compose.yaml it relates to is somewhat helpful. I agree that we should iterate on file layout in the repo though. Let's hold off on that for this go round, if that's OK?